plugins/mentions/plugin.js

```diff
@@ -417,14 +409,29 @@
 	 * ```javascript
 	 * var definition = {
 	 * 		feed: feed,
-	 * 		template: '<li data-id={id}><img src="{iconSrc}" alt="{name}">{name}</li>'
+	 * 		viewTemplate: '<li data-id={id}><img src="{iconSrc}" alt="{name}">{name}</li>'
 	 * }
-	 *
 	 * ```
 	 *
 	 * @property {String} [template=CKEDITOR.plugins.autocomplete.view.itemTemplate]
 	 */
 
+	/**
+	 * Output template used to write selected match into an editor.
```

Comment:
Again to keep it DRY I'd suggest smartly link it to the existing property API docs, that is driven by this configuration.